docs/diagrams/UiClassDiagram.puml

```diff
@@ -30,15 +31,18 @@ HiddenOutside ..> Ui
 
 UiManager .left.|> Ui
 UiManager -down-> "1" MainWindow
+
 MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
 MainWindow *-down-> "1" PersonListPanel
 MainWindow *-down-> "1" StatusBarFooter
 MainWindow --> "0..1" HelpWindow
 
 PersonListPanel -down-> "*" PersonCard
+PersonListPanel -down-> "*" ContactDetails
```

Comment:
Should this be `0..1` instead? Since PersonListPanel only holds either null or one instance of the panel